examples/preact/ops.js

```diff
@@ -25,7 +25,8 @@ const ModulePromise = initModule();
 export function rethrowIfCritical(err) {
   // If it's precisely Error, it's a custom error; anything else - SyntaxError,
   // WebAssembly.RuntimeError, TypeError, etc. - is treated as critical here.
-  if (err.constructor !== Error) {
+  if (err.constructor !== Error)
+  {
```

Comment:
Nit: what happened to formatting here and below with braces moving onto new lines? Can you fix manually or run them through Prettier please?

Reply:
Also looks like those functions are now duplicated with camera class? Can we reuse them instead? (by updating the demo)

Reply:
We _can_ update the preact demo (and probably should), but since I'm very much not experienced with React ~and thoroughly dislike working with it~ I kind of wanted to leave that up to you ;)

I made the changes to the repo in such a way that the demo wouldn't be broken precisely for this reason